README.md

```diff
@@ -21,6 +21,25 @@ Note: there is also an online version, which can be found at [my-mind.github.io]
   * ~~Open index.html in your webbrowser~~ currently not working
 * Done! If need be, you can find the manual [here](https://github.com/ondras/my-mind/wiki)
 
+## Docker
+To dockerize My Mind 
+
+1. Clone the repo 
+   
+   `git clone git@github.com:ondras/my-mind.git`
+
+2. Build the image
+   
+   `docker build -t my-mind .`
+
+3. Run container
+   
+   `docker run -d --name=my-mind -p 80:80 my-mind`
+
```

Comment:
port 80 is often taken you might want to use port 8080:80 and then write down that this could be changed